bridgev2/portal.go

```diff
@@ -4046,7 +4046,16 @@ func (portal *Portal) syncParticipants(
 		return fmt.Errorf("failed to get current members: %w", err)
 	}
 	delete(currentMembers, portal.Bridge.Bot.GetMXID())
-	powerChanged := members.PowerLevels.Apply(portal.Bridge.Bot.GetMXID(), currentPower)
+	// Ensure baseline power levels match initial room creation defaults.
+	// See createMatrixRoomInLoop: we enforce bot at 9001 and lock down
```

Comment:
Is this right/safe?

Reply:
It's unsafe for rooms that aren't created by the bridge and doesn't seem necessary for local bridges either